Makefile

```diff
@@ -73,10 +71,12 @@ ifeq ("$(PGO_BASEOS)", "centos8")
 endif
 
 DEBUG_BUILD ?= false
-GCFLAGS=
+GO_BUILD = $(GO_CMD) build
```

Comment:
Any chance we might still be able to allow for `install` vs `build`?

I currently leverage this in conjunction with my `GO_BIN` to quickly/easily launch our various Operator applications as part of my typical development process, since `GO_BIN` is already part of my default`PATH` (with this change I suppose I could also make `$PGOROOT/bin` part of my standard `PATH`).

Reply:
I don't like `install` because it creates things outside the project directory.

I use `./bin/pgo` to run the client, but (most of) the other binaries go into subdirectories of `./bin` which I can see being a hassle. I'd like to move those up into `./bin` so that `$PGOROOT/bin` can be added to `PATH` as you suggest. Do you know of anything besides the Dockerfiles that would be affected by that?

Reply:
So, most executables are named the same as a non-empty subdirectory in `bin`. I can move `postgres-operator` up alongside `pgo` and `apiserver` at least. Would that alleviate most of the pain?

Reply:
I am definitely good with that approach.  With `pgo`, `postgres-operator` and `apiserver` being the most common binaries run via my `PATH`, this should definitely alleviate most pain (I am also fine with this approach considering that some of the various other binaries such as `pgo-scheduler`, `pgo-rmdata` & `pgo-backrest` will likely change in some way or another moving forward, e.g. `pgo-backrest` will soon be moved to the Container Suite project as part of compaction).

Reply:
Done.